src/handlers/Handlers.h

```diff
@@ -0,0 +1,44 @@
+#include <handlers/Context.h>
+#include <handlers/Status.h>
+
+#include <unordered_map>
+#include <functional>
+#include <iostream>
+
+#ifndef RIPPLE_REPORTING_HANDLERS_H
+#define RIPPLE_REPORTING_HANDLERS_H
+
+namespace RPC 
+{
+
+// Maximum and minimum supported API versions
+// Defaults to APIVersionIfUnspecified
+constexpr unsigned int APIVersionIfUnspecified = 1;
+constexpr unsigned int ApiMinimumSupportedVersion = 1;
+constexpr unsigned int ApiMaximumSupportedVersion = 1;
+constexpr unsigned int APINumberVersionSupported =
+    ApiMaximumSupportedVersion - ApiMinimumSupportedVersion + 1;
+
+static_assert(ApiMinimumSupportedVersion >= APIVersionIfUnspecified);
+static_assert(ApiMaximumSupportedVersion >= ApiMinimumSupportedVersion);
+
+// Indicates the level of administrative permission to grant.
+// NOTE, this does not currently affect RPCs in clio, and will be implimented later.
+enum class Role { GUEST, USER, IDENTIFIED, ADMIN, PROXY, FORBID };
```

Comment:
Since the roles are not being used, let's not add them now

Reply:
I think there are only two roles, and the `DOSGuard` handles them. There are clients who are whitelisted, and clients who aren't.